README.md

```diff
@@ -25,6 +25,10 @@ or
 docker build -t owasp/modsecurity-crs .
 docker run -p 80:80 -ti -e PARANOIA=4 -e PROXY=1 --rm owasp/modsecurity-crs
 ```
+
+## Apache
+The Apache webserver is configured via the `httpd-default.conf` file overriding directives from the base file.
```

Comment:
Using a file called `httpd-default.conf` in `/usr/local/apache2/conf/extra/` is a non-standard pattern. I don't think this is a good idea.

We should make it easy for developers knowing _Apache_ (but not this specific setup) to find configuration details where they expect them. A hint in the README is okay as an idea, but sticking to standard patterns would be superior, IMHO.

Other opinions? @mhutter @srueg

Reply:
As discussed: we'll add our own config file `/usr/local/apache2/conf/extra/httpd-modsecurity.conf` which includes all our config changes (i.e. using env vars). That way we still use the defaults of the base image.